examples/3d-tiles/app.js

```diff
@@ -106,28 +152,28 @@ export default class App extends PureComponent {
     this.setState({viewState});
   }
 
-  _renderLayers() {
-    const {tilesetJson, tilesetUrl} = this.state;
+  _renderLayer() {
+    const {layerProps} = this.state;
     return (
-      tilesetJson &&
+      layerProps &&
       new Tileset3DLayer({
         id: 'tileset-layer',
-        tilesetJson,
-        tilesetUrl,
+        ...layerProps,
```

Comment:
Nit: In my opinion using `layerProps` saves some typing but makes the code a little harder to read.

To be able to scan this layer creation and see exactly which props are being passed in is very helpful.

Now I have to start looking for what is in `layerProps`.